src/quo2/components/colors/color_picker/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.components.colors.color-picker.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [reagent.core :as reagent]
+            [quo2.components.colors.color-picker.style :as style]))
+
+(defn picker-colors
+  [blur?]
+  (concat (map (fn [color]
+                 {:name  color
+                  :color (colors/custom-color-by-theme color (if blur? 60 50) 60)})
+               [:blue :yellow :turquoise :copper :sky :camel :orange :army :pink :purple :magenta])
```

Comment:
nit: separate color list into it's own def

```
(def colors-list [:blue :yellow :turquoise :copper :sky :camel :orange :army :pink :purple :magenta])

(defn picker-colors
  [blur?]
  (concat (map (fn [color]
                 {:name  color
                  :color (colors/custom-color-by-theme color (if blur? 60 50) 60)})
              colors-list )
          [{:name            :fengshui
            :color           (colors/theme-colors (colors/custom-color :feng 50)
                                                  (colors/custom-color :chui 50))
            :secondary-color (colors/theme-colors (colors/custom-color :chui 50)
                                                  (colors/custom-color :feng 50))}]))
```

Reply:
Done